internal/services/authorization/role_management_policy_resource.go

```diff
@@ -0,0 +1,1097 @@
+						Computed:      true,
+						ConflictsWith: []string{"activation_rules.0.required_conditional_access_authentication_context"},
+					},
+
+					"require_justification": {
+						Description: "Whether a justification is required during activation",
+						Type:        pluginsdk.TypeBool,
+						Optional:    true,
+						Computed:    true,
+					},
+
+					"require_ticket_info": {
+						Description: "Whether ticket information is required during activation",
+						Type:        pluginsdk.TypeBool,
+						Optional:    true,
+						Computed:    true,
+					},
+				},
+			},
+		},
+
+		"notification_rules": {
+			Description: "The notification rules of the policy",
+			Type:        pluginsdk.TypeList,
+			Optional:    true,
+			Computed:    true,
+			MaxItems:    1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"active_assignments": {
+						Description: "Notifications about active assignments",
+						Type:        pluginsdk.TypeList,
+						Optional:    true,
+						Computed:    true,
+						MaxItems:    1,
+						Elem: &pluginsdk.Resource{
+							Schema: notificationRuleSchema(),
+						},
+					},
+					"eligible_activations": {
+						Description: "Notifications about activations of eligible assignments",
+						Type:        pluginsdk.TypeList,
+						Optional:    true,
+						Computed:    true,
+						MaxItems:    1,
+						Elem: &pluginsdk.Resource{
+							Schema: notificationRuleSchema(),
+						},
+					},
+					"eligible_assignments": {
+						Description: "Notifications about eligible assignments",
+						Type:        pluginsdk.TypeList,
+						Optional:    true,
+						Computed:    true,
+						MaxItems:    1,
+						Elem: &pluginsdk.Resource{
+							Schema: notificationRuleSchema(),
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func (r RoleManagementPolicyResource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Description: "The name of the policy",
+			Type:        pluginsdk.TypeString,
+			Computed:    true,
+		},
+
+		"description": {
+			Description: "The Description of the policy",
+			Type:        pluginsdk.TypeString,
+			Computed:    true,
+		},
+
+		"display_name": {
+			Description: "The display name of the policy",
+			Type:        pluginsdk.TypeString,
+			Computed:    true,
+		},
+	}
+}
+
+func (r RoleManagementPolicyResource) Create() sdk.ResourceFunc {
+	return sdk.ResourceFunc{
+		Timeout: 30 * time.Minute,
+		Func: func(ctx context.Context, metadata sdk.ResourceMetaData) error {
+			client := metadata.Client.Authorization.RoleManagementPoliciesClient
+
+			var config RoleManagementPolicyModel
+			if err := metadata.Decode(&config); err != nil {
+				return fmt.Errorf("decoding %+v", err)
+			}
+
+			id := parse.NewRoleManagementPolicyID(config.Scope, "")
+			roleDefinitionId := config.RoleDefinitionId
+
+			scopedId, err := getScopedPolicyId(ctx, metadata, id, roleDefinitionId)
+			if err != nil {
+				return err
+			}
```

Comment:
I'm not sure I follow why two parsers are being used here when `parse.RoleManagementPolicyId{}` and `rolemanagementpolicies.RoleManagementPolicyId{}` appear to have the same structure?

Can we not use `rolemanagementpolicies.RoleManagementPolicyId{}` (i.e. `scopedId`) as the resource ID rather than the `Id` field in the response, which seems to change on every update to the policy?